catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -223,6 +236,8 @@ public Metacard transform(InputStream input, String id)
             }
 
             String metacardContentType = metacard.getContentTypeName();
+            metacard.setAttribute(new AttributeImpl(Core.DATATYPE, getDatatype(metacardContentType)));
```

Comment:
Should we do this as a PreIngestPlugin instead? Otherwise you could have some InputTransformers that set this field and some that don't.